controllers/database_controller.go

```diff
@@ -549,6 +544,44 @@ func (r *DatabaseReconciler) createProxy(ctx context.Context, dbcr *kciv1alpha1.
 	return nil
 }
 
+func (r *DatabaseReconciler) createConnectionString(ctx context.Context, dbcr *kciv1alpha1.Database) error {
+	// First of all the password should be taken from secret because it's not stored anywhere else
+	databaseSecret, err := r.getDatabaseSecret(ctx, dbcr)
+	if err != nil {
+		return err
+	}
+	// Then parse the secret to get the password
+	databaseCred, err := parseDatabaseSecretData(dbcr, databaseSecret.Data)
+	if err != nil {
+		return err
+	}
+	// Fill ConnectionStringFields struct
+	dbUrl := ConnectionStringFields{
+		DatabaseHost: dbcr.Status.ProxyStatus.ServiceName,
+		DatabasePort: dbcr.Status.ProxyStatus.SQLPort,
+		UserName:     dbcr.Status.UserName,
+		Password:     databaseCred.Password,
+		DatabaseName: dbcr.Status.DatabaseName,
+	}
+	// Generate the connection string
+	dbConnectionString, err := generateConnectionString(dbcr, dbUrl)
+	if err != nil {
+		return err
+	}
+	// Update database-credentials secret.
+	if connectionString := databaseCred.ConnectionString; connectionString == dbConnectionString {
```

Comment:
I think it can be just like `databaseCred.ConnectionString == dbConnectionString`

Reply:
Huh, it's pretty dumb yeah. Fixed now